### ISSUE TITLE
Upgrade mikepenz icons dependency, without the icons disappearing

### ISSUE THREAD

**Issue by oliverlockwood** [cyclestreets/android#473]
See #471 and #472.

**Comment by HilaryN** [comment]
I know you've self-assigned this but I just want to share my findings so far (though no harm in you confirming as I could have made a mistake):

- The icons disappear when you upgrade from 5.0.3 to 5.1.0 (i.e. prior to 5.2.8)
- The menu option is there, even though you can't see it - you can tap on it and LiveRide will start (luckily, as I only realised the icons were missing when I was part-way through a journey and wanted to start LiveRide when I got to the bit of route I didn't know!)
- I reproduced the same thing by creating an Empty (Hello World) Activity app with an options menu copied from CycleStreets.  Icons were there when I used 5.0.3 (though strangely I had to rotate the emulator screen back and forth before they would appear) and disappeared when I changed to 5.2.8.
- No difference between Android 9 and 10.
- I can get the icons to appear using IconicsDrawable function, but I don't understand why this needs to be used.  It's as though the "app:ico"... xml attributes are being ignored.
- I tried comparing IconicsMenuInflaterUtil.kt 5af2820 and 031c523 (5.0.3 --> 5.1.0) but couldn't see any changes.

**Comment by oliverlockwood** [comment]
You are welcome to assign this to yourself if you'd like to work on it. Not likely to be an imminent priority for me! 

My first step would be to look through the release notes / changes documented in the iconics library itself, to see if I could find clues.

**Comment by HilaryN** [comment]
Yes, I tried looking through the notes, but nothing jumped out at me :-(

**Comment by HilaryN** [comment]
https://github.com/mikepenz/Android-Iconics/issues/580#issuecomment-851341207
